scala/opentransit/src/main/scala/com/azavea/opentransit/database/StopBuffersTable.scala

```diff
@@ -34,6 +34,10 @@ object StopsBuffersTable extends Logging {
 
   def stopsBufferTable = TableQuery[StopsBuffers]
 
+  def bufferRadius(db: DatabaseDef): Double = db withSession { implicit session =>
+    stopsBufferTable.first.radius
```

Comment:
I didn't want to pass the request all the way through, but i ended up doing so anyway, so this can be removed.